### DISCUSSION TITLE
support zh-TW & zh-HK

### DISCUSSION THREAD

**Discussion by sofent11** [rainbow-me/rainbowkit#1738] [Ideas]
request support support zh-TW & zh-HK in rainbowkit-beta

**Comment by Chen-Hsien** [comment]
true!

**Comment by DanielSinclair** [comment]
Would `zh-Hant` suffice for generic Traditional Chinese characters, or would you expect significant technical dialect differences between these regions?

**Comment by Chen-Hsien** [comment]
There will be a slight difference between TW and HK, but it’s not a big deal. 
Using zh-Hant is fine initially. In Taiwan, users generally don't like to see zh-CN on the web.

**Comment by DanielSinclair** [comment]
These locales are now supported in [v2.1.5](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%402.1.5)
